src/widget/contentlayout.cpp

```diff
@@ -71,15 +71,11 @@ void ContentLayout::clear()
     QLayoutItem* item;
     while ((item = mainHead->layout()->takeAt(0)) != 0)
     {
-        item->widget()->hide();
-        item->widget()->setParent(nullptr);
```

Comment:
Did you try build qtox with this commit?
This lines are needed, otherwise it break UI.